### PR TITLE
MicroVU: Fix status of flag info register

### PR DESCRIPTION
**Pull request by ssakash**
Removed a part of code which looks like a hack ,  please do some testing on whether it causes/fixes any issues related to the vector units,  thanks. 

**Do not merge the pull request, only  intended for testing purposes**

Edit: someone ought to give me a medal for creating pull requests from my smartphone :P

### CODE REVIEW

**Comment by MrCK1** [comment]
I'll test Wipeout Fusion and some other games in a few hours

**Comment by gregory38** [comment]
Did you seach on github conversation on this kind of code. I remember vaguely something for a similar hack.

File: pcsx2/x86/microVU_Flags.inl [hunk 1]
```diff
@@ -232,8 +232,8 @@ __fi void mVUsetFlags(mV, microFlagCycles& mFC) {
 	}
 
 	mVUregs.flagInfo |= ((__Status) ? 0 : (xS << 2));
-	mVUregs.flagInfo |= ((__Mac||1) ? 0 : (xM << 4));
-	mVUregs.flagInfo |= ((__Clip)   ? 0 : (xC << 6));
+	mVUregs.flagInfo |= ((__Mac) ? 0 : (xM << 4));
+	mVUregs.flagInfo |= ((__Clip) ? 0 : (xC << 6));
```
Comment (gregory38) [inline pcsx2/x86/microVU_Flags.inl]: Vertical alignment was nicer.

**Comment by ssakash** [comment]
I didn't seem to find anything useful on the conversation. I'll fix the vertical alignment later.

Though honesty we need more test cases on how the following change influences other games. It's possible that this hack is used to circumvent around a part which is not accurately emulated.

**Comment by MrCK1** [comment]
(I guess my comment didn't post earlier) 

Wipeout and Jak1 worked fine. Gran Turismo 4 seemed to stutter and also crashed 4 times. I'll edit this later with the 3 logs I had prepared yesterday :)

**Comment by ssakash** [comment]
About Gran Turismo 4 : I think it's related to another issue where the renderer allocates lots of useless framebuffers which In turn increases memory overhead and causes crashes. Could you reproduce the crash on native/custom resolution ?

**Comment by MrCK1** [comment]
I'll try to duplicate this/check memory usage again later. I don't recall seeing any major spikes this time (which is the usual case for GT4 crashes) but maybe I'm wrong :P

**Comment by ssakash** [comment]
@MrCK1 

Any update on the status of the issue ?

**Comment by MrCK1** [comment]
Sorry, forgot about this. My GPU has changed since the last report (750>970) but I wasn't able to duplicate any issues in GT4. Most of its issues present are GS related anyways.

**Comment by ssakash** [comment]
Thanks for the test. though I'm not planning on getting this merged unless it actually **fixes** something ;)

**Comment by FlatOutPS2** [comment]
What kind of fixes are possible with this?

**Comment by refractionpcsx2** [comment]
> What kind of fixes are possible with this?

I'm expecting none, but I'm also hoping that there are no breakages either, this is technically more "right"

**Comment by FlatOutPS2** [comment]
I tested several games, no new issues or fixes. I did see a potential regression in Ridge Racer V, but it turns out that setting rendering threads to 3(or higher) in software mode creates graphical glitches in that game that are not related to the this PR.

**Comment by gregory38** [comment]
Does it impact the perf ?

**Comment by refractionpcsx2** [comment]
> Does it impact the perf ?

I doubt it, if it does it will be in the positive direction.

**Pull request closed without merging**